engine/src/flutter/shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -267,6 +267,13 @@ bool FlutterWindowsEngine::Run(std::string_view entrypoint) {
   std::string executable_name = GetExecutableName();
   std::vector<const char*> argv = {executable_name.c_str()};
   std::vector<std::string> switches = project_->GetSwitches();
+  if (project_->impeller_enabled()) {
+    switches.push_back("--enable-impeller=true");
+  } else if (std::find(switches.begin(), switches.end(),
+                       "--enable-impeller=true") != switches.end()) {
+    switches.push_back("--enable-impeller=false");
+  }
```

Comment:
> The behavior we've used so far is that the flutter run flag overrides the project flag, since the run flag is usually for testing and we don't want folks to be continually recompiling.

Isn't the code here doing the opposite of that? It looks like `project_->impeller_enabled()` wins; isn't that the project flag set at compile time? I would expect the code for what you are describing to only look at `embeller_enabled()` if no switch started with `--enable-impeller=`.

Reply:
Yeah let me switch these around.